nav_msgs/Path path
---
nav_msgs/Path generated_path
nav_msgs/Path executed_path
---
